src/client/components/Dropdown/DropDown.css

```diff
@@ -0,0 +1,15 @@
+select {
+  display: block;
+  width: 400px;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  border-radius: 2rem;
+  border: 1px solid #f6f6f6;
+  background-color: #f6f6f6;
```

Comment:
Prabin it doesn't match with the one in Figma. Font style, colour and dropdown nothing matched. 
<img width="717" alt="Screenshot 2020-11-26 at 14 06 36" src="https://user-images.githubusercontent.com/61879499/100354538-9bf34500-2ff0-11eb-83ec-14d21faad2df.png">

In our figma link there is details about everything. Dropdown should be like Polygon.

<img width="1180" alt="Screenshot 2020-11-26 at 14 05 59" src="https://user-images.githubusercontent.com/61879499/100354478-8c73fc00-2ff0-11eb-81a5-c247d59eea5e.png">

Reply:
Another thing I would like to mention. Please link your PR with the issue you are doing. It helps to find the issue and the PR, also help to review that. Please do that with all your PR. Thank you. 

<img width="349" alt="Screenshot 2020-11-26 at 14 18 39" src="https://user-images.githubusercontent.com/61879499/100355567-49b32380-2ff2-11eb-886e-6ab94ac92e36.png">

Reply:
Good comments @Pallabi-ghosh